frontend/src/components/Login.js

```diff
@@ -1,75 +1,163 @@
-                  <div className='register-button' >
-                    <Button onClick={ ()=> navigate('./admin')} type='submit' color="primary" variant='contained' >Login</Button>
-                  </div>
+              <div align="center" className="register-sign">
+                <h1>Login or Sign up</h1>
+                <div className="register-fields">
+                  <TextField
+                    error={emailError}
+                    helperText={
+                      emailError ? "Please enter a valid email adress" : ""
+                    }
+                    inputProps={{ style: { color: "black" } }}
+                    color="secondary"
+                    margin="normal"
+                    className="register-text"
+                    label="Email Address"
+                    variant="outlined"
+                    required
+                    value={email}
+                    onChange={(e) => {
+                      setEmail(e.target.value);
+                      validateEmail(e.target.value);
+                    }}
+                  />
+                  <TextField
+                    inputProps={{ style: { color: "black" } }}
+                    color="secondary"
+                    margin="normal"
+                    className="register-text"
+                    label="Password"
+                    variant="outlined"
+                    type="password"
+                    required
+                    value={password}
+                    onChange={(e) => {
+                      setPassword(e.target.value);
+                    }}
+                  />
+                </div>
 
-                  <Typography className='register-link' sx={{margin:3,fontSize: "1.2rem"}}> Do you have an account ?
-                    <Link onClick={ ()=> setSwtichSignup(true)} sx={{margin:1, color: "white", fontSize: "1.4rem", fontWeight: "700" }} className="signup-style">
-                       Sign up
-                    </Link>
-                  </Typography> 
-
-                  
+                <div className="register-button">
+                  <p>{loginStatusMessage}</p>
+                  <Button
+                    onClick={handleClick}
+                    type="submit"
+                    color="primary"
+                    variant="contained"
+                  >
+                    Login
+                  </Button>
+                </div>
 
+                <Typography
+                  className="register-link"
+                  sx={{ margin: 3, fontSize: "1.2rem" }}
+                >
+                  {" "}
+                  Do you have an account ?
+                  <Link
+                    onClick={() => setSwtichSignup(true)}
+                    sx={{
+                      margin: 1,
+                      color: "white",
+                      fontSize: "1.4rem",
+                      fontWeight: "700",
+                    }}
+                    className="signup-style"
```

Comment:
please dont use className for styling, use sx and js objects in separate file or styles from our library

for text size use variant